ERROR_CODES.md

```diff
@@ -0,0 +1,84 @@
+# Error Codes for BPP
+
+## ID: 
+BECKN-RFC-034
+
+## Draft ID
+Draft-01
+
+## Title:
+Error Codes
+
+## Category:
+Network Policy
+
+## Status:
+Protocol Draft
+
+## Published on:
+January 21, 2022
+
+## Expires on:
+January 20, 2023 or Date of publication of next draft which ever is earlier
+
+## License:
+CC-BY-ND
+
+## Authors:
+1. Ravi Prakash : ravi@becknfoundation.org
+
+## Reviewers:
+1. Sujith Nair : sujith@becknfoundation.org
+2. Pramod Varma : pramod@ekstep.org
+
+## To Comment
```

Comment:
Can we remove this section as people can comment directly on Github? I had added this section basis the mailing lists that IETF used to maintain for publishing RFCs. But I don't think that is necessary anymore as Github is pretty sophisticated in itself.